shell_scripts/coq_platform_override_dev_pkg.sh

```diff
@@ -15,8 +15,7 @@ do
     pkg="${override%%=*}"
     url="${override#*=}"
     echo "Overriding $pkg.dev url with $url"
-    mkdir -p opam/opam-coq-archive/extra-dev/packages/$pkg/$pkg.dev/
-    opam show --raw $pkg.dev | tr -s '\n' ' ' | sed "s@ src: [^ ]*@ src: \"$url\"@" > opam/opam-coq-archive/extra-dev/packages/$pkg/$pkg.dev/opam
+    opam pin add -y --kind=http $pkg.dev $url
```

Comment:
I am not sure I like the -y. Coq Platform usually has set "OPAMYES=1", but when creating a pick manually I don't want any -y, because it could mess up things - I want it to stop then if there is something to ask.

Reply:
Ok, but be aware that `opam pin` must necessary use `-y` in many cases, so I'm not sure that'd be useful for you.